XcodeServerSDK/Utils/HTTPUtils.swift

```diff
@@ -92,7 +90,7 @@ public class HTTP {
             }
             
             
-        }).resume()
+        })?.resume()
```

Comment:
Interesting, `dataTaskWithRequest` changed to returning an optional? In that case, I think it's a good place for another guard (we want to still call the completion block if `nil` is returned, otherwise it would never be called).

So higher up, try `guard let task = self.session.dataTaskWithRequest(...) else {` and call completion block with some error in the else block. Then, underneath all that, start the task with `task.resume()`. Does that make sense?